staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -877,7 +884,10 @@ func (t crdConversionRESTOptionsGetter) GetRESTOptions(resource schema.GroupReso
 			c,
 			&unstructuredCreator{},
 			crdserverscheme.NewUnstructuredObjectTyper(),
-			&unstructuredDefaulter{delegate: Scheme},
+			&unstructuredDefaulter{
+				delegate:          Scheme,
+				structuralSchemas: t.structuralSchemas,
```

Comment:
need to set `structuralSchemaGK` here?

Reply:
possibly so... `TestCustomResourceDefaulting` integration test is failing

Reply:
yes, this was it. Don't trust changes after 11pm.